css/css-flexbox/flex-minimum-width-flex-items-014.html

```diff
@@ -6,32 +6,20 @@
 <style>
 .min-content {
     width: 10px;
-    min-width: -webkit-min-content;
-    min-width: -moz-min-content;
-    min-width: -ie-min-content;
-    min-width: -o-min-content;
```

Comment:
The `-o` prefix isn't necessary any more since Opera switched to webkit almost 10 years ago 😛